include/simdjson/inline/document_stream.h

```diff
@@ -100,14 +100,34 @@ static inline size_t trimmed_length_safe_utf8(const char * c, size_t len) {
 
 namespace simdjson {
 namespace dom {
-
+///Os3
 really_inline document_stream::document_stream(
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR. Remove the ///0s3 comment

Reply:
done